streamparse/thrift.py

```diff
@@ -0,0 +1,569 @@
+"""
+Python Storm Topology DSL
+"""
+
+import io
+
+import six
+import thriftpy
+
+# Apache storm.thrift file as string to simplify loading it.
+_THRIFT_STR = """
```

Comment:
`__file__` can help you find package data. Is there a reason otherwise why you want to embed as a string?